.github/workflows/tests.yml

```diff
@@ -25,15 +23,15 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: '3.9'
-    - name: Install dependencies
-      run: |
-        python -m pip install -r requirements.txt
     - name: Install Aurora
       run: ${{ matrix.build_command }}
     - name: Move to root and try importing Aurora
       run: |
         cd /
         python -c "import aurora; print(aurora.__version__)"
+    - name: Run test_basic.py
+      run: |
+        python examples/test_basic.py
```

Comment:
@fsciortino let me know in case there are other scripts that would be better suited as CI tests

Reply:
@eguiraud in principle, all the "test_*" scripts were created with the idea of using pytest, but I only ever did that on the command line, rather than online with proper CI. Anyway, I think that using just test_basic.py would be enough -- it catches most of what we want to check.

Reply:
`test_basic.py` in particular does not seem to be written as a pytest test: it has no test function inside, all instructions are at global scope, it does its own simple command line parsing...?